src/x/instrument/config_test.go

```diff
@@ -256,7 +262,7 @@ func newConfiguration() MetricsConfiguration {
 
 func assertMetricsEqual(t *testing.T, listener net.Listener, expected map[string]xjson.Map) {
 	url := fmt.Sprintf("http://%s/metrics", listener.Addr().String()) //nolint
-	resp, err := http.Get(url) //nolint
+	resp, err := http.Get(url)                                        //nolint
```

Comment:
what's the lint here?

Reply:
I think the preferred method is `http.NewRequestWithContext` and then calling `client.Do(req)`. The lint here is that we're making an HTTP call w/o a context. We're less concerned about that in tests, so you'll see `//nolint`s when this is done. This could've been more precise with `//nolint:noctx`